app/views/admin/users/_current_roles.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="crayons-card p-6 grid gap-2">
+  <h2>Current Roles</h2>
+  <p>(View full history in notes below)</p>
+  <div class="crayons-card__body">
+    <ul>
+      <% @user.roles.each do |role| %>
+        <% if role.name == "banned" %>
+          <li>Suspended</li>
+        <% elsif role.name == "super_admin" || @user.id == current_user.id %>
+          <li><%= role.name.titlecase %></li>
+        <% else %>
+          <li>
+            <%= role.name.titlecase %> <em><%= role.resource_name.to_s %></em> &nbsp;
+            <%= link_to "REMOVE", url_for(action: :destroy, user_id: @user.id, role: role.name.to_sym), method: :delete, data: { confirm: "Are you sure?" },
+                                                                                                        class: "crayons-btn crayons-btn--danger crayons-btn--s lh-base fs-xs" %>
```

Comment:
Open to suggestions on how to clean this up a bit if anyone has any! I tried to break this line up a bit to make it more readable and to help it fall within our target line length.

Reply:
A small nit pick—I know the RFC mentioned that the button should read “REMOVE”, but I find that very hard to read and a bit verbose when it repeats over and over the way we have it now. Perhaps just an “X” here instead, or perhaps we can use a icon that already exists in our app? I think this RFC was accepted before we had the concept of product weigh in/approval, otherwise I would’ve weighed in here sooner! But this should be an easy change 😅

Reply:
@vaidehijoshi After looking through the rest of the admin dashboard, it looks like we use "Destroy" and "Remove" on the buttons used to remove badge achievements, HTML variants, navigation links, etc., so  I opted to go with an "X" in place of "REMOVE" for the role removal buttons per your suggestion!